source/deployment/bulk-loading-data-format.rst

```diff
@@ -490,10 +474,120 @@ Fields of the User object
+      <kbd>"mention"</kbd> - Only for mentions.<br>
+      <kbd>"none"</kbd> - Never.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">desktop_duration</td>
+      <td valign="middle">string</td>
+      <td>Preference for how long desktop notifications remain on screen. Must be one of the following values:<br>
+      <kbd>"3"</kbd> - 3 seconds.<br>
+      <kbd>"5"</kbd> - 5 seconds.<br>
+      <kbd>"10"</kbd> - 10 seconds.<br>
+      <kbd>"unlimited"</kbd> - Unlimited.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">desktop_sound</td>
+      <td valign="middle">string</td>
+      <td>Preference for whether desktop notification sound is played. Must be one of the following values:<br>
+      <kbd>"on"</kbd> - Sound is played.<br>
+      <kbd>"off"</kbd> - Sound is not played.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">email</td>
+      <td valign="middle">string</td>
+      <td>Preference for email notifications. Must be one of the following values:<br>
+      <kbd>"immediately"</kbd> - Email notifications are sent immediately.<br>
+      <kbd>"15"</kbd> - Email notifications are grouped and sent every 15 minutes. Requires email batching to be enabled.<br>
+      <kbd>"60"</kbd> - Email notifications are grouped and sent every 1 hour. Requires email batching to be enabled.<br>
+      <kbd>"never"</kbd> - Email notifications are not sent.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">mobile</td>
+      <td valign="middle">string</td>
+      <td>Preference for sending mobile push notifications. Must be one of the following values:<br>
+      <kbd>"all"</kbd> - For all activity.<br>
+      <kbd>"mention"</kbd> - Only for mentions.<br>
+      <kbd>"none"</kbd> - Never.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">mobile_push_status</td>
+      <td valign="middle">string</td>
+      <td>Preference for when push notifications are triggered. Must be one of the following values:<br>
+      <kbd>"online"</kbd> - When online, away or offline.<br>
+      <kbd>"away"</kbd> - When away or offline.<br>
+      <kbd>"offline"</kbd> - When offline.</td>
+      <td align="center" valign="middle">No</td>
+      <td align="center" valign="middle">No</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">channel</td>
+      <td valign="middle">string</td>
+      <td>Whether @all, @channel and @here trigger mention notifications???/td>
```

Comment:
Supported values are "true" for whether to trigger on them or "false" to not. Note that this is string values, not a boolean.